### DISCUSSION TITLE
add favorites

### DISCUSSION THREAD

**Discussion by sebastiensoulac** [punxaphil/custom-sonos-card#319] [Q&A] [answered]
How do I add favorites?
Do favorites go back to Sonos favorites?
I would like to have my radios in the all favorites, what should I do?

**Comment by punxaphil** [comment, marked as answer]
It must be added to my Sonos https://support.sonos.com/en/article/add-favorites-to-my-sonos

**Comment by sebastiensoulac** [comment]
Thanks for the feedback.
I have my favorites 
![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/379163b0-ce5e-43e4-af1f-f5d757d0e3c9)
I wanted to know if it's normal not to have the radio logos on the player that plays?

**Reply by punxaphil** [reply]
check readme on http to https

**Reply by sebastiensoulac** [reply]
Thank you for your feedback @punxaphil , but unfortunately I can't find where to look.

Can you tell me what I need to modify in the interface in order to have the radio logo on the listening player?

**Reply by punxaphil** [reply]
The image is not available in Home Assistant as far as I know. Only the current songs image.
However, you can use `mediaChannelEquals` to set a hardcoded image. More details in the readme.

**Comment by sebastiensoulac** [comment]
I've added this to the card code for testing with the image you have in the Readme source code.
`type: custom:sonos-card
artworkAsBackground: true
sections:
  - player
  - volumes
  - groups
  - grouping
  - media browser
predefinedGroups: []
topFavorites:
  - ''
hideGroupCurrentTrack: true
dynamicVolumeSlider: true
showVolumeUpAndDownButtons: false
showNonSonosPlayers: false
mediaArtworkOverrides:
  - mediaTitleEquals: FG DISCO FUNK
    imageUrl: >-
      https://mytuner.global.ssl.fastly.net/media/tvos_radios/2BDTPrpMbn_cTdteqo.jpg
entities: []
excludeItemsInEntitiesList: true`

As you can see, I didn't understand what I was supposed to do.
Can you help me?
Thanks again.

**Reply by punxaphil** [reply]
that is hard to read
please format as code

**Comment by sebastiensoulac** [comment]
`type: custom:sonos-card
artworkAsBackground: true
sections:
  - player
  - volumes
  - groups
  - grouping
  - media browser
predefinedGroups: []
topFavorites:
  - ''
hideGroupCurrentTrack: true
dynamicVolumeSlider: true
showVolumeUpAndDownButtons: false
showNonSonosPlayers: false
mediaArtworkOverrides:
  - mediaTitleEquals: FG DISCO FUNK
    imageUrl: >-
      https://mytuner.global.ssl.fastly.net/media/tvos_radios/2BDTPrpMbn_cTdteqo.jpg
entities: []
excludeItemsInEntitiesList: true`

**Reply by sebastiensoulac** [reply]
Sorry, I seem to have put the code in <>.
in addition to an image.
![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/cfcae65b-aef2-4247-a652-d761471e002e)

**Reply by punxaphil** [reply]
You need to play some media

**Comment by sebastiensoulac** [comment]
I would like to see the logo of the radio station currently playing.

I select my radio station in my favorites and when I switch to the player I want the logo of the radio station currently playing.

**Reply by punxaphil** [reply]
sure, but provide a better screenshot. The screenshot above isn't playing anything

**Comment by sebastiensoulac** [comment]
Ok sorry

radio FG selected in favorites
![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/fd99a2d8-6823-4293-ba80-d1bff199ef48)

radio currently playing
![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/19250f55-f9de-4640-83d2-242c5413ebf8)
Unfortunately I don't have the FG radio logo as in my favorites.

I hope this helps

**Comment by punxaphil** [comment]
Any errors in the web console?

**Comment by sebastiensoulac** [comment]
I don't get any error messages in the debug.
For your information:
I've entered the address of my sonos in my config file.
I changed the name of my sonos today and it doesn't change anything on the HA interface.

**Comment by sebastiensoulac** [comment]
I don't know if this will help you. 

![image](https://github.com/punxaphil/custom-sonos-card/assets/51950939/eb4d618b-0be3-4700-bf9b-4a7907abba9d)

**Comment by punxaphil** [comment]
Try `mediaChannelEquals` instead

**Reply by punxaphil** [reply]
Ignore that, I see you don't have that in your attributes. 
But `mediaContentIdEquals` should work?
